zproject/default_settings.py

```diff
@@ -22,6 +22,15 @@
 # prod_settings_template.py, and in the initial /etc/zulip/settings.py on a new
 # install of the Zulip server.
 
+################
+# Public Listing Of Realms.
+#
+# If you need people to be able to list their realms on the
+# communities page, you should set the OSS_SHOWCASE to True.
+#
+# To avoid spam currently only sponsored realms can be listed publically.
+OSS_SHOWCASE = False
+
```

Comment:
This is placed in a very odd place in this file; I'm going to push a revised version of this commit that moves it (and same with the other settings).  I'd also like us to rename this to `CUSTOMER_SHOWCASE_ENABLED`, for more consistency with the naming of other settings.